_posts/francois_gerthoffert/2019-03-03-ZenCrepes-GitHub-Agile-Analytics-made-easy.md

```diff
@@ -0,0 +1,301 @@
+---
+layout: post
+title:  "Introducing ZenCrepes - Agile analytics across GitHub orgs & repos"
+breadcrumb: true
+author: francois_gerthoffert
+date: 2019-03-03
+categories: francois_gerthoffert
+tags:
+    - Agile
+    - GitHub
+    - Sprints
+teaser:
+    info: Have you ever tried managing a scrum team across GitHub organizations and repositories? ZenCrepes has been created to facilitate project management for teams operating solely over GitHub issues, and across multiple organizations & repositories.
+    image: francois_gerthoffert/zencrepes/zencrepes-issues.png
+header:
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+Approximately a year ago, our team transitioned away from Jira to GitHub for our ticketing system, bringing our source code and project management in the same place. A few months back, I wrote a [blog post detailing this transition](https://softeng.oicr.on.ca/francois_gerthoffert/2018/10/02/Transitioning-away-from-Jira-Lessons-learnt/) and another one giving some [insights about some Agile metrics that could be used for team](https://softeng.oicr.on.ca/francois_gerthoffert/2018/10/20/Agile-another-take-on-estimates/) management, in particular in the context of research. 
+
+Both articles contained screenshots from an “unknown tool” and now is the right time to start introducing ZenCrepes (bonus points for those who guess where the name is coming from), a personal project I’ve been spending evenings on for the past few months to provide Agile analytics and management across GitHub organizations & repositories.
```

Comment:
Would be fun if you gave the explanation later at the end of the article?  I read through and didn't see any answer :).  Unless you want to keep it secret or have people guess.  Maybe add it at the end as a bonus or post-script?  Up to you!